homeassistant/components/fritzbox/sensor.py

```diff
@@ -67,7 +67,7 @@ def suitable_nextchange_time(device: FritzhomeDevice) -> bool:
 
 def suitable_temperature(device: FritzhomeDevice) -> bool:
     """Check suitablity for temperature sensor."""
-    return device.has_temperature_sensor and not device.has_thermostat
+    return bool(device.has_temperature_sensor)
```

Comment:
the property is not typed in the lib, so we need to cast it explicit as bool to make mypy happy:
```
homeassistant/components/fritzbox/sensor.py:70: error: Returning Any from function declared to return "bool" 
```

Reply:
Maybe we should fix that 😉

Reply:
yeah, but it needs to be fixed in the lib 😬